apps/pipelines/migrations/0006_pipelinechathistory_pipelinechatmessages.py

```
@@ -0,0 +1,44 @@
+        ('experiments', '0094_consentform_working_version_and_more'),
+        ('pipelines', '0005_auto_20240802_0039'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='PipelineChatHistory',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+                ('updated_at', models.DateTimeField(auto_now=True)),
+                ('type', models.CharField(choices=[('node', 'Node History'), ('named', 'Named History'), ('global', 'Global History'), ('none', 'No History')], max_length=10)),
```

Ruff

Line too long (176 > 120)

(E501)